### PR TITLE
pseduo stripe integration still not pulling in correct data, still ne…

### PR DESCRIPTION
**Pull request by Tulf**
…ed to integrate wwith backend, mostly commiting to update branch with master

### CODE REVIEW

**Comment by Tulf** [comment]
Seems like the auth0 login isn't working on the preview, but it's very likely because the URL is blocked by google's API. I made no changes that should affect that at least.

**Review by MichaelNey — APPROVED** [review]

**Comment by MichaelNey** [comment]
> Seems like the auth0 login isn't working on the preview, but it's very likely because the URL is blocked by google's API. I made no changes that should affect that at least.

Yep. The deployment previews have unique URL's, so the Auth0 CORS approval doesn't apply to it. No worries.

**Pull request merged**
